applications/matter_weather_station/src/app_task.cpp

```diff
@@ -99,8 +99,15 @@ bool sHaveBLEConnections;
 
 LedState sLedState = LedState::kAlive;
 
-Identify sIdentify = { chip::EndpointId{ kIdentifyEndpointId }, AppTask::OnIdentifyStart,
-		       AppTask::OnIdentifyStop, EMBER_ZCL_IDENTIFY_IDENTIFY_TYPE_AUDIBLE_BEEP };
+/* Add identify for all endpoints */
+Identify sIdentifyRoot = { chip::EndpointId{ kIdentifyEndpointRootId }, AppTask::OnIdentifyStart, AppTask::OnIdentifyStop,
```

Comment:
I think Identify is not needed on the root endpoint. Endpoints 1,2,3 should be sufficient.